src/controllers/webhookController.ts

```
@@ -398,6 +398,88 @@ export async function handleFlutterwaveWebhook(
+      503,
+      ErrorCodes.CONFIG_ERROR,
+    );
+  }
+
+  const rawBody = (req as unknown as { rawBody?: Buffer }).rawBody;
+  if (!rawBody || !Buffer.isBuffer(rawBody)) {
+    throw new AppError(
+      "Raw body required for verification",
+      400,
+      ErrorCodes.RAW_BODY_REQUIRED,
+    );
```

ESLint

[error] Replace `⏎······"Raw·body·required·for·verification",⏎······400,⏎······ErrorCodes.RAW_BODY_REQUIRED,⏎····` with `"Raw·body·required·for·verification",·400,·ErrorCodes.RAW_BODY_REQUIRED`

(prettier/prettier)